cmd/docs.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/speakeasy-api/speakeasy/internal/config"
+	"github.com/speakeasy-api/speakeasy/internal/docsgen"
+	"github.com/speakeasy-api/speakeasy/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
+	Short: "Use this command to generate content, compile, and publish SDK docs.",
+	Long:  "Use this command to generate content, compile, and publish SDK docs. This feature is currently in closed beta, please reach out to speakeasy for more information.",
+	RunE:  utils.InteractiveRunFn("Provide a docs sub command"),
+}
+
+var genDocsContentCmd = &cobra.Command{
+	Use:   "content",
+	Short: "Use this command to generate content for the SDK docs directory.",
+	Long:  "Use this command to generate content for the SDK docs directory.",
+	RunE:  genSDKDocsContent,
+}
+
+func docsInit() {
+	rootCmd.AddCommand(docsCmd)
+	// SDK Docs Content flags.
+	genDocsContentCmd.Flags().StringP("out", "o", "", "path to the output directory")
+	genDocsContentCmd.MarkFlagRequired("out")
+	genDocsContentCmd.Flags().StringP("schema", "s", "./openapi.yaml", "local filepath or URL for the OpenAPI schema")
+	genDocsContentCmd.MarkFlagRequired("schema")
+	genDocsContentCmd.Flags().StringP("langs", "l", "", "a list of languages to include in SDK Doc generation")
```

Comment:
probably should add an example like `go,java` or whatever the expected format is

Reply:
Good point